src/client/api/events/sub_serialized_output.c

```diff
@@ -4,29 +4,45 @@
 #include <string.h>
 
 #include "client/api/events/sub_serialized_output.h"
+#include "core/address.h"
+#include "core/models/payloads/tagged_data.h"
+#include "core/utils/macros.h"
 
 int event_sub_txn_included_msg(event_client_handle_t client, int *mid, char const transaction_id[], int qos) {
-  if ((strlen(transaction_id)) != EVENT_TXN_ID_LEN) {
+  if ((strlen(transaction_id)) != BIN_TO_HEX_BYTES(IOTA_TRANSACTION_ID_BYTES)) {
```

Comment:
One parentheses too many.
```suggestion
  if (strlen(transaction_id) != BIN_TO_HEX_BYTES(IOTA_TRANSACTION_ID_BYTES)) {
```

Reply:
Thanks Matjaz.